spec-tests/src/test/java/org/candlepin/spec/imports/ImportErrorSpecTest.java

```diff
@@ -0,0 +1,208 @@
+/**
+ * Copyright (c) 2009 - 2022 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.spec.imports;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatNoException;
+import static org.candlepin.spec.bootstrap.assertions.StatusCodeAssertions.assertBadRequest;
+import static org.candlepin.spec.bootstrap.assertions.StatusCodeAssertions.assertConflict;
+
+import org.candlepin.dto.api.client.v1.ImportRecordDTO;
+import org.candlepin.dto.api.client.v1.OwnerDTO;
+import org.candlepin.dto.api.client.v1.PoolDTO;
+import org.candlepin.dto.api.client.v1.UpstreamConsumerDTO;
+import org.candlepin.dto.api.client.v1.UserDTO;
+import org.candlepin.spec.bootstrap.assertions.OnlyInStandalone;
+import org.candlepin.spec.bootstrap.client.ApiClient;
+import org.candlepin.spec.bootstrap.client.ApiClients;
+import org.candlepin.spec.bootstrap.client.SpecTest;
+import org.candlepin.spec.bootstrap.data.builder.Consumers;
+import org.candlepin.spec.bootstrap.data.builder.Export;
+import org.candlepin.spec.bootstrap.data.builder.ExportGenerator;
+import org.candlepin.spec.bootstrap.data.builder.Owners;
+import org.candlepin.spec.bootstrap.data.util.Importer;
+import org.candlepin.spec.bootstrap.data.util.UserUtil;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@SpecTest
+@OnlyInStandalone
+public class ImportErrorSpecTest {
+
+    private ApiClient admin;
+    private OwnerDTO owner;
+    private ApiClient userClient;
+    private Importer importer;
+
+    @BeforeEach
+    public void beforeAll() {
+        admin = ApiClients.admin();
+        owner = admin.owners().createOwner(Owners.random());
+        UserDTO user = UserUtil.createUser(admin, owner);
+        userClient = ApiClients.basic(user);
+        userClient.consumers().createConsumer(Consumers.random(owner));
```

Comment:
This is creating a consumer and not using it anywhere?

Reply:
Done.